Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -558,6 +558,7 @@
     <ConditionalEmbeddedResource Include="VSPackage.resx">
       <MergeWithCTO>true</MergeWithCTO>
       <LogicalName>VSPackage.Resources</LogicalName>
+      <SubType>Designer</SubType>
```

Comment:
what is this change?

Reply:
That change is unintentional and was added by VS. Removed.